src/mlpack/methods/reinforcement_learning/sac.hpp

```diff
@@ -0,0 +1,178 @@
+/**
+ * @file methods/reinforcement_learning/sac.hpp
+ * @author Nishant Kumar
+ *
+ * This file is the definition of SAC class, which implements the
+ * soft actor-critic algorithm.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_RL_SAC_HPP
+#define MLPACK_METHODS_RL_SAC_HPP
+
+#include <mlpack/prereqs.hpp>
+
+#include "replay/random_replay.hpp"
+#include <mlpack/methods/ann/activation_functions/tanh_function.hpp>
+#include <mlpack/methods/ann/loss_functions/mean_squared_error.hpp>
+#include <mlpack/methods/ann/visitor/parameters_visitor.hpp>
+#include "training_config.hpp"
+
+namespace mlpack {
+namespace rl {
+
+/**
+ * TODO: Add citation
```

Comment:
Let's add this now

Reply:
I keep forgetting these small things. Makes me realize the importance of reviewing. Thanks again for pointing it out :+1: